cloud/store/service.go

```diff
@@ -161,11 +169,44 @@ func (st *Service) Execute(req *command.ApplyRequest) error {
 	if leader == "" {
 		return ErrLeaderNotFound
 	}
-	_, err := st.rpc.apply(leader, req)
+	_, err := st.rpc.Apply(leader, req)
 	return err
 }
 
-func removeNilTenants(tenants []*command.Tenant) []*command.Tenant {
+func (s *Service) Join(ctx context.Context, id, addr string, voter bool) error {
+	log.Printf("membership.join %v %v %v", id, addr, voter)
```

Comment:
Just a reminder that go now has the [log/slog](https://pkg.go.dev/log/slog) pacakge 🙂

Reply:
Great that you brought that up . We'II definitely explore go new logging capabilities, especially for the cloud package